bowlingScore.js

```diff
@@ -0,0 +1,18 @@
+class BowlingGame {
+ constructor() {
+   this.frames = []
```

Comment:
I think the way your test works this might be an array of all rolls rather than frames. Frames may require 2 (or even 3) balls, so can be a bit difficult to decide at this stage.